src/OAuth2.ts

```diff
@@ -37,6 +37,13 @@ const REQUIRED_SCOPES = [
     "bot",
 ];
 
+const PUPPET_SCOPES = [ // See https://stackoverflow.com/a/28234443
+    "identify",
+    "read",
+    "post",
```

Comment:
From what I can tell, the `client` scope grants basically everything - wee-slack and I believe emacs-slack both only request client. Do we actually need these scopes?

Reply:
I guess not :)